website/content/docs/platform/k8s/injector/examples.mdx

```diff
@@ -385,3 +385,36 @@ spec:
 
 [pkiCert]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#pkicert
 [writeToFile]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#writeToFile
+
+## Cross namespace secret sharing ((#cross-namespace))
+
+1. Configure Vault for secret sharing across namespaces as described in the
+   [How-to configure cross namespace access in Vault Enterprise][cross-namespace]
+   Knowledge base article.
```

Comment:
```suggestion
1. [Configure Vault for secret sharing across namespaces][cross-namespace].
```
We can simplify this step since the instructions are essentially the same as the document title